app/templates/scripts/helper-functions.sh

```diff
@@ -217,6 +217,26 @@ function enable_development_modules {
   echo
 }
 
+##
+# Convert csv migration files to sql tables.
+##
+function convert_csv_to_sql {
+  echo -e "${LBLUE}> Converting csv migration files to sql tables${RESTORE}"
+  cd $ROOT/www
+  CSV2SQL=`drush | grep "csv2sql"`
+  INSTALL_PROFILE= `drush vget install_profile`
+
+  if [ ! "$CSV2SQL" ]; then
+    drush dl csv2sql
+  fi
+
+  csv_files="profiles/$INSTALL_PROFILE/modules/custom/$INSTALL_PROFILE_migrate/csv/*/*.csv"
```

Comment:
You don't need to change the profile, you can use ``skeleton`` and it will be renamed to the profile name by yo